content/_events/2020/april.md

```diff
@@ -0,0 +1,126 @@
+---
+published: false
+upcoming: true
+title: Documentation templates for fun and profit
+event-id: april-2020-documentation-templates
+year: 2020
+event-date: Thursday, April 23, 2020
+locale: san-francisco
+location: google-345-spear-street
+rsvp_open: true
+description: |
+Find out how templates make better documentation.
+form:
+  id: id (don't include the #, for the modal dialog)
+  title: title for the form's modal dialog
+  embed: |
+   embed code for the form
+agenda: |
+    * 6:30 p.m. Socialize and network
+    * 7 p.m. Presentation, followed by questions
+    * 7:30 p.m. to 8:15 p.m. Work in groups
+    * 8:15 to 8:45 p.m. Socialize and pack up
+    * 9 p.m. Lights out
+
+## Documentation templates for fun and profit
```

Comment:
We should get rid of both of these headings, because the Title in the front matter already displays at the top of the page, so this would just be a duplicate line. We either don't need to start with a heading at all, or we should jump into a tagline with an `<h2>` (Don't use `<h1>` - that's for the title only).